e2e-tests/cypress/support/before-you-start-appellant-selects-the-site/select-site-option.js

```diff
@@ -0,0 +1,31 @@
+import {
+  getListedBuildingOption,
+  getMajorDwellingsOption,
+  getMajorGeneralIndustryStorageOrWarehousingOption,
+  getMajorRetailAndServicesOption,
+  getMajorTravellingAndCaravanPitchesOption,
+  getNoneOfTheseOption,
+} from '../page-objects/appellant-selects-the-site-po';
+
+export const selectSiteOption = (option) =>{
+   switch (option){
+     case 'None of these':
+       getNoneOfTheseOption().click();
```

Comment:
While click may solve our purpose, this does not ensure that the checkbox is checked (i.e. It will uncheck previously selected checkbox). An appropriate one would be check()
```suggestion
       getNoneOfTheseOption().check();
```

Reply:
Updated as per review comments